docs/source/using/managing-minishift.adoc

```diff
@@ -293,23 +293,32 @@ CONTAINER    IMAGE                   COMMAND                CREATED        STATU
 71fe8ff16548 openshift/origin:v1.5.1 "/usr/bin/openshift s" 4 minutes ago  Up 4 minutes  origin
 ----
 
-[[experimental-functionality]]
-== Experimental functionality
+[[experimental-features]]
+== Experimental Features
 
-To allow users and developers to get early access to some upcoming features, we offer a way to enable experimental functionality.
+If you want to get early access to some upcoming features and experiment, you can enable some of those in Minishift.
 
-You do this by setting the environment variable `MINISHIFT_ENABLE_EXPERIMENTAL` which will make additional flags available:
+[IMPORTANT]
```

Comment:
Could we move this under the code listing 'export ...' It really breaks my flow in reading and understanding this, when it sits between these two sentences.